README.md

```diff
@@ -1,20 +1,122 @@
+- Data Explorer: explore your dataset by feature sets and other metrics such as predicted Y or true Y
+- Model Statistics: explore the distribution of your model outcomes and performance metrics
+- Interpretability: view the aggregate and individual feature importances across your model and dataset
+- Counterfactual Example What-If's (new): create automatically generated diverse sets of counterfactual examples for each datapoint that is minimally perturbed in order to switch its predicted class or output. Also create your own counterfactual datapoint by perturbing feature values manually to observe the new outcome of your model prediction.
+- Causal Analysis (new): view the aggregate and individual causal effects of *treatment features* (features which you are interested in controlling to affect the outcome) on the outcome in order to make informed real-life business decisions. See recommended treatment policies for segmentations of your population for features in your dataset to see its effect on your real-life outcome. 
 
-# CLI support for Responsible AI
+This new feature offers users a new powerful and robust toolkit for understanding your model and data in order to develop your machine learning models responsibly, now all in one place and integrated with your AzureML workspace.
 
-## Create a Model Analysis using the CLI
-Responsible AI job creation in the CLI is based on .YAML config files that you as a user can create. We have provided samples of these .YAML files that you can update with your specific settings. Follow this quickstart guide to create Responsible AI jobs with the CLI
-- [How to create an Responsible AI job with the CLI](https://github.com/RachelKellam/RAI-vNext-Preview/blob/main/docs/cli/how-to-create-model-analysis-cli.md)
+❗ **Please note:** This initial version of the Responsible AI toolbox currently does not support the integration of fairness metrics. For fairness metrics, please refer to our existing offering [here.](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-machine-learning-fairness-aml)
 
-# Python SDK Support for Responsible AI
-- [How to create an Responsible AI job with the Python SDK](https://github.com/Azure/RAI-vNext-Preview/blob/main/docs/PythonSDK/how-to-create-model-analysis-python.md)
+## Supported scenarios, models and datasets
+
+`azureml-responsibleai` supports computation of Responsible AI insights for `scikit-learn` models that are trained on `pandas.DataFrame`. The `azureml-responsibleai` accept both models and pipelines as input as long as the model or pipeline implements a `predict` or `predict_proba` function that conforms to the `scikit-learn` convention. If not compatible, you can wrap your model's prediction function into a wrapper class that transforms the output into the format that is supported (`predict` or `predict_proba` of `scikit-learn`), and pass that wrapper class to modules in `azureml-responsibleai`.
```

Comment:
The second sentence has issues....

- "The `azureml-responsibleai` accept' is missing a noun
- I thought that the `azureml-responsible` package only knew about models, not pipelines?

Reply:
SciKit Learn pipelines. Fixing.